src/data/STATUSES/layers/patch5.3.ts

```diff
@@ -6,5 +6,11 @@ export const patch530: Layer<StatusRoot> = {
 	data: {
 		// 5.3 GNB Changes - Brutal Shell updated duration
 		BRUTAL_SHELL: {duration: 30},
+
+		// 5.3 AST Changes - Neutral Sect (Noct)
+		DIURNAL_BALANCE: {
+			name: 'Diurnal Balance (Benefic)',
+		},
+
```

Comment:
Note to self, this is gonna conflict with WAR changes, whichever is more ready for merge can go in and Ay will feex the other.